FoodRecipes/View/recipeCell.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  recipeCell.swift
+//  FoodRecipes
+//
+//  Created by Mac on 26/05/2023.
+//
+
+import UIKit
+
+class recipeCell: UITableViewCell {
+    @IBOutlet weak var imgRecipe: UIImageView!
+    
+    @IBOutlet weak var lblServings: UILabel!
+    @IBOutlet weak var btnFavIcon: UIButton!
+    @IBOutlet weak var lblChefName: UILabel!
+    @IBOutlet weak var lblRecipeName: UILabel!
+    
+    @IBOutlet weak var lblFoodType: UILabel!
+    
```

Comment:
As a naming convention, we set the name first and then its type, for example here instead of `imgRecipe` it can be `recipeImage`, or `recipeImg`. Another example, instead of `lblServings` we can use `servingsLabel`